src/build-application.js

```diff
@@ -228,7 +236,17 @@ module.exports = async function buildApplication(watch = false, minify = false)
       } else {
         console.log(chalk.yellow(`+ ${cmdString} browser client "${clientName}"`));
 
-        const inputFile = path.join(cwd, 'src', 'clients', clientName, 'index.js');
+        // Note: We first try to read the index.js file, if it doesn't exist we try the .ts version
+        var inputFile = path.join(cwd, 'src', 'clients', clientName, 'index.js');
+        try {
+          if (!fs.existsSync(inputFile)) {
+            //console.log(chalk.gray("Note: index.js does not exist, trying .ts file..."))
+            inputFile = path.join(cwd, 'src', 'clients', clientName, 'index.ts');
+          }
+        } catch(err) {
+          console.error(err)
+        }
```

Comment:
Note: This part was "hardcoded" before with the `index.js` - maybe there is a option to skip the hardcoded dependency completely here, but I don't understand the build script enough to make sure that I don't drop any edge cases here - maybe you have an idea.